src/lib/modernExtend.ts

```diff
@@ -526,6 +527,57 @@ export function forcePowerSource(args: {powerSource: 'Mains (single phase)' | 'B
     return {configure, isModernExtend: true};
 }
 
+export interface QuirkAddEndpointClusterArgs {
+    endpointID: number, inputCluster?: string | number, outputCluster?: string | number,
```

Comment:
We should allow passing a list here

Reply:
I also considered that yes, but I opted to keep the code simpler and just add more to the extend: [] block on the device.

I'll change it to accept a list and rewrite the function tonight.

Reply:
Looks like I at least got better at typescript lately, the difference is not that big.